src/components/hv-image/index.ts

```diff
@@ -24,13 +25,12 @@ export default class HvImage extends PureComponent<HvComponentProps> {
 
   static localNameAliases = [];
 
-  props: HvComponentProps;
-
   render() {
     const { skipHref } = this.props.options || {};
-    const imageProps = {};
-    if (this.props.element.getAttribute('source')) {
-      let source = this.props.element.getAttribute('source');
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const imageProps: Record<string, any> = {};
```

Comment:
Should we be using 'unknown' instead of 'any' in these places?

Reply:
we'll want to revisit everywhere where we silenced the `@typescript-eslint/no-explicit-any` - I think these might indicate there's a slightly bigger refactor needed.